tests/Xamin/HandlebarsTest.php

```diff
@@ -438,6 +438,30 @@ function ($template, $context, $arg) use ($that) {
         $this->setExpectedException('InvalidArgumentException');
         $engine->getHelpers()->call('invalid', $engine->loadTemplate(''), new \Handlebars\Context(), '', '');
     }
+	
+	public function testRegisterHelper()
+    {
+        $template = '{{#grand test "test2"}}  In 1: {{test4}} {{#grand ../test \'test3\'}} In 2: {{test5}}{{#parent}}  In 3: {{test6}}  {{../../../test}}{{/parent}}{{/grand}}{{/grand}}';
+        $vars = array('test' => 'Hello World');
+        $expected = '  In 1: Hello World  In 2: This is Test 5  In 3: This is Test 6  Hello World';
```

Comment:
@cblanquera could you make this test a bit more clear and maybe split it into specific scenarios that you support like hash parameters? Because aside from better test coverage, most of the time, we referrer users to this file to look for examples.

Also please use space instead of tab for indentation.